patchdeps.py

```diff
@@ -113,8 +124,8 @@ def print_depends(patches, depends):
             continue
         print("%s depends on: " % p)
         for dep in patches:
-            if dep in depends[p]:
-                desc = getattr(depends[p][dep], 'desc', None)
+            if dependency := dependencies.get(dep):
```

Comment:
In commit " refactor: Convert Bunch into an Enum ", this adds a reference to `dependencies`, which is not introduced until commit " style: ´print_depends` "